app/src/main/java/org/simple/clinic/plumbing/infrastructure/MixpanelInfrastructure.kt

```diff
@@ -1,31 +0,0 @@
-package org.simple.clinic.plumbing.infrastructure
-
-import android.app.Application
-import com.mixpanel.android.mpmetrics.MixpanelAPI
-import org.simple.clinic.BuildConfig
-import org.simple.clinic.appconfig.Country
-import org.simple.clinic.appconfig.Deployment
-import org.simple.clinic.user.User
-import javax.inject.Inject
-
-class MixpanelInfrastructure @Inject constructor(
-    private val application: Application
-) : Infrastructure {
-
-  private val mixpanel = MixpanelAPI.getInstance(application, BuildConfig.MIXPANEL_TOKEN, true)
-
```

Comment:
Can we remove the `BuildConfig.MIXPANEL_TOKEN` from gradle.properties file too?  Also we would need to remove it from CI pipelines as well.